frontend/pages/shelter/pet/[id].tsx

```diff
@@ -1,36 +1,57 @@
+import { Animal } from "@contract";
```

Comment:
updates to this file are to facilitate removal of PetData from enum/types and to use Animal.Attributes from contract instead - in preparation of getting data from API